src/runtime-tools/linux/install-dotnet.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
don't you need to call these from setup.sh and setup.ps1 ?

Reply:
Ok! That's way simpler I'll make the updates